azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/AddMojo.java

```diff
@@ -158,18 +161,14 @@ protected void doExecute() throws Exception {
 
     //region Load all templates
     protected BindingTemplate loadBindingTemplate(String type) throws Exception {
-        info("");
-        info(LOAD_TEMPLATES);
-
         try (final InputStream is = AddMojo.class.getResourceAsStream("/bindings.json")) {
             final String bindingsJsonStr = IOUtil.toString(is);
             final BindingsTemplate bindingsTemplate = new ObjectMapper()
                 .readValue(bindingsJsonStr, BindingsTemplate.class);
-            info(LOAD_TEMPLATES_DONE);
             return bindingsTemplate.getBindingTemplateByName(type);
         } catch (Exception e) {
-            error(LOAD_TEMPLATES_FAIL);
-            throw e;
+            warning(LOAD_BINDING_TEMPLATES_FAIL);
+            return null;
```

Comment:
Why we swallow the error?

Reply:
BindingTemplate only contains default value and prompt message, addMojo could also work without it, so just swallow this error and show warning here.